src/fixate/ui_gui_qt/ui_gui_qt.py

```diff
@@ -738,96 +738,32 @@ def _topic_UI_action(self, msg, callback_obj):
             callback_obj.set_target_finished_callback(self.sig_button_reset.emit)
             self.sig_choices_input.emit(msg, ("Fail",))
 
-    def _topic_UI_req(self, msg, q):
-        """
-        This can be replaced anywhere in the project that needs to implement the user driver
-        The result needs to be put in the queue with the first part of the tuple as 'Exception' or 'Result' and the
-        second part is the exception object or response object
-        :param msg:
-         Message for the user to understand what to do
-        :param q:
-         The result queue of type queue.Queue
-        :return:
-        """
+    def _topic_UI_req(self, msg):
         if self.closing:
-            q.put("Result", None)
             return
-        self.gui_user_input(msg, ("Continue",))
-        q.put("Result", None)
 
-    def _topic_UI_req_choices(self, msg, q, choices, target, attempts=5):
-        """
-        This can be replaced anywhere in the project that needs to implement the user driver
-        Temporarily a simple input function.
-        The result needs to be put in the queue with the first part of the tuple as 'Exception' or 'Result' and the
-        second part is the exception object or response object
-        This needs to be compatible with forced exit. Look to user action for how it handles a forced exit
-        :param msg:
-         Message for the user to understand what to input
-        :param q:
-         The result queue of type queue.Queue
-        :param target:
-         Optional
-         Validation function to check if the user response is valid
-        :param attempts:
-        :return:
-        """
+        self._gui_user_input(msg, ("Continue",))
+
+    def _topic_UI_req_choices(self, msg, q, choices):
         if self.closing:
+            # I don't think the result of this code ever gets used, nothing looking for "ABORT_FORCE"
+            # unpacks from a tuple. This can probably be deleted.
```

Comment:
2 things in main look for this, I really don't understand what these are doing though. Doesn't make sense to me that we would inidicate a forced abort during a user input function